docs/tutorials/projects/filtered-camera.md

```diff
@@ -103,18 +103,17 @@ For more information see [Configure data capture](/data/capture/#configure-data-
 
 ## The machine learning (ML) model for filtering
 
-Now that your camera component is steadily capturing images to the Viam app, you can train a machine learning (ML) model on those images to be able to detect certain objects in the camera feed.
```

Comment:
I feel that somewhere we must still discuss why we use ML models, regardless of whether they are trained or uploaded or added (module):

> to be able to detect certain objects in the camera feed.

is still useful to me to understand what the next ~120 lines discuss.

Reply:
Added a suggestion of where I think it could fit